src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-server.cpp

```diff
@@ -0,0 +1,645 @@
+        for (auto & transportConnection : mCurrentConnections)
+        {
+            if (transportConnection.fabricIndex == ctx.mCommandHandler.GetAccessingFabricIndex())
+            {
+                transportConnection.transportConfiguration.transportStatus = transportStatus;
+                connectionIDList.push_back(transportConnection.transportConfiguration.connectionID);
+            }
+        }
+    }
+    else
+    {
+        TransportConfigurationStorageWithFabricIndex * transportConfiguration = FindStreamTransportConnection(connectionID.Value());
+        if (transportConfiguration == nullptr)
+        {
+            ChipLogError(Zcl, "HandleSetTransportStatus: ConnectionID Not Found.");
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::NotFound);
+            return;
+        }
+        if (transportConfiguration->fabricIndex != ctx.mCommandHandler.GetAccessingFabricIndex())
+        {
+            ChipLogError(Zcl, "HandleSetTransportStatus: ConnectionID Not Found.");
+            ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::NotFound);
+            return;
+        }
+
+        transportConfiguration->transportConfiguration.transportStatus = transportStatus;
+        connectionIDList.push_back(connectionID.Value());
+    }
+    // Call the delegate
+    status = mDelegate.SetTransportStatus(connectionIDList, transportStatus);
+    ctx.mCommandHandler.AddStatus(ctx.mRequestPath, status);
+}
+
+void PushAvStreamTransportServer::HandleManuallyTriggerTransport(
+    HandlerContext & ctx, const Commands::ManuallyTriggerTransport::DecodableType & commandData)
+{
+    Status status           = Status::Success;
+    uint16_t connectionID   = commandData.connectionID;
+    auto & activationReason = commandData.activationReason;
+    auto & timeControl      = commandData.timeControl;
+
+    TransportConfigurationStorageWithFabricIndex * transportConfiguration = FindStreamTransportConnection(connectionID);
+
+    if (transportConfiguration == nullptr)
+    {
+        ChipLogError(Zcl, "HandleManuallyTriggerTransport: ConnectionID Not Found.");
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::NotFound);
+        return;
+    }
+
+    if (transportConfiguration->fabricIndex != ctx.mCommandHandler.GetAccessingFabricIndex())
+    {
+        ChipLogError(Zcl, "HandleManuallyTriggerTransport: ConnectionID Not Found.");
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::NotFound);
+        return;
+    }
+
+    if (mDelegate.GetTransportStatus(connectionID) == PushAvStreamTransportStatusEnum::kBusy)
+    {
+        ChipLogError(Zcl, "HandleManuallyTriggerTransport: Connection is Busy");
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::Busy);
+        return;
+    }
+
+    if (transportConfiguration->transportConfiguration.transportStatus == TransportStatusEnum::kInactive)
+    {
+        auto clusterStatus = static_cast<uint8_t>(StatusCodeEnum::kInvalidTransportStatus);
```

Comment:
```suggestion
        auto clusterStatus = to_underlying(StatusCodeEnum::kInvalidTransportStatus);
```
I'm going to stop commenting on this.  Just fix them all.

Reply:
Fixed